www/dufs/Portfile

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        sigoden dufs 0.43.0 v
+revision            0
```

Comment:
please add "github.tarball_from" and set it to the appropriate value: "releases (preferred if upstream provides the), otherwise "archive". Doing so will likely require you to update the "checksums".